core/src/main/scripts/import_data_validator/validateData.py

```diff
@@ -567,7 +566,10 @@ def validate(self):
                 return
 
             # parse the start-of-file comment lines
-            self.processTopLines(top_comments)
+            if not self.processTopLines(top_comments):
+                self.logger.info(
+                    'Invalid header comments, skipped data in file')
```

Comment:
All the hook methods I added this notification to return whether they have
logged any errors that make the file unparseable.
On 13 Jan 2016 18:34, "pieterlukasse" <notifications@github.com> wrote:

> In core/src/main/scripts/import_data_validator/validateData.py
> <https://github.com/thehyve/cbioportal/pull/1#discussion_r49622037>:
>
> > @@ -567,7 +566,10 @@ def validate(self):
> >                  return
> >
> >              # parse the start-of-file comment lines
> > -            self.processTopLines(top_comments)
> > +            if not self.processTopLines(top_comments):
> > +                self.logger.info(
> > +                    'Invalid header comments, skipped data in file')
>
> is this not related to the clinical data file? Maybe it should be moved to
> this specific validator instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thehyve/cbioportal/pull/1/files#r49622037>.
>